exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -580,11 +580,11 @@ func (b *BTCMarkets) SubmitOrder(ctx context.Context, s *order.Submit) (*order.S
 		return nil, err
 	}
 
-	if s.Side == order.Sell {
-		s.Side = order.Ask
+	if s.Side.IsLong() {
+		s.Side = order.Buy
 	}
-	if s.Side == order.Buy {
-		s.Side = order.Bid
+	if s.Side.IsShort() {
+		s.Side = order.Sell
```

Comment:
```suggestion
		s.Side = order.Ask
```

To match API expectation:

<img width="678" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/4685270/a2ab7377-cd6a-4564-ba66-212287d369ed">

Reply:
solved